game/templates/main-menu.html

```diff
@@ -41,18 +45,43 @@ <h1>Riddle Me This</h1>
     {% if user.is_authenticated %}
     <div id="mySidenav" class="sidenav">
         <a href="javascript:void(0)" class="closebtn" onclick="closeNav()"><i class="fa fa-window-close"></i></a>
-        <a class="sidetext" style="color: #effeff;"> Welcome, {{user.name}}</a>
-        <form method="POST" > {% csrf_token %}
-            {{ form.as_p }}
-        </form>
-        <div id="block_container">
-            <a class="log_btn" id="button" href="{% url 'logout' %}"><span>Change Name</span></a>
-            <a id='logout-bth' class="log_btn" href="{% url 'logout' %}"><span>Logout</span></a>
+        {% if user.name == 'guest'%}
+        <p style="color: #effeff; text-align: center;">No Profile..!</p>
+        {% else %}
+        <p style="color: #effeff; text-align: center;">Player Profile:</p>
+        <img class="provider_pic" src="{{social_user.get_avatar_url}}">
+        <a class="sidetext" style="color: #effeff;"> Username: {{ social_user.extra_data.given_name }} {{social_user.extra_data.first_name}}            
+            <button id="edit-name" style="padding: 0; border: none; background: none;">
+                <i class="fa fa-pencil-square" style="color: #effeff;" onclick="editName()"></i>
+            </button>
+            <br>
+            Email: {{ social_user.extra_data.email }}
```

Comment:
Since we already have username and log in provider shown, so I think user's email is not necessary now.

Reply:
Removed. Thanks 🙇